### PR TITLE
Add Translation Portuguese of Brazil

### PR DESCRIPTION
**Pull request by JNylson**

### CODE REVIEW

File: src/lang/pt_BR.rc [hunk 1]
```diff
@@ -0,0 +1,1611 @@
...
+        MENUITEM "&Testar", ID_TEST
+        MENUITEM SEPARATOR
+        MENUITEM "E&xcluir\tDel", ID_DELETERES
+        MENUITEM SEPARATOR
+        MENUITEM "ID de Ass&ociação...", ID_IDASSOC
+        MENUITEM "&Macros Predefinidas...", ID_PREDEFMACROS
+        MENUITEM "Configurações de &Fonte...", ID_FONTS
+        MENUITEM "Definir Camin&hos...", ID_SETPATHS
+        MENUITEM SEPARATOR
+        MENUITEM "Selecionar Idioma da Inte&rface (UI语言选择)...", ID_CHOOSEUILANG
```
Comment (katahiromz) [inline src/lang/pt_BR.rc:89]: Don't translate this line, please.

**Review by katahiromz — CHANGES_REQUESTED** [review]

**Comment by JNylson** [comment]
Fixed requested string

File: src/lang/pt_BR.rc (hunk 1 above)
Comment (JNylson) [reply src/lang/pt_BR.rc]: ok, i'm fixed requested string

**Comment by katahiromz** [comment]
Thanks.

**Pull request merged**